### PR TITLE
Adedd lab8

### PR DESCRIPTION
**Pull request by UzhvenkoJulia**
Static arrays. Multidimensional arrays

### CODE REVIEW

File: cw8.c [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            printf("The sum is: %.2lf\n", sum);  /*else {
+            printf("m and n should be <= %d!\n", MAX_LARGE);*/
+        }  // return sum;
+        // }
+
+        case 7: {
+            int n;
+            printf("Task 7: Enter n (the size of the square matrix): ");
+            scanf("%d", &n);
+            double matrix7[n][n];
```
Comment (mecchmatProjects) [inline cw8.c]: Not cross platforming

File: cw8.c [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+            input_matrix(m, n, matrix3);
+            print_matrix(m, n, matrix3, 0);
+            break;
+        }
+
+        case 4: {
+            int m, n;
+            printf("Task 4: Enter m and n: ");
+            scanf("%d %d", &m, &n);
+            double matrix4[m][n];
```
Comment (mecchmatProjects) [inline cw8.c]: not cross platform

File: cw8.c (hunk 1 above)
Comment (UzhvenkoJulia) [reply cw8.c]: Fixed dynamic declaration of arrays based on input sizes.

**Review by mecchmatProjects — COMMENTED** [review]
Could You give me some test results?

**Comment by UzhvenkoJulia** [comment]
Added tests to tasks

**Review by mecchmatProjects — APPROVED** [review]
will do

**Pull request merged**